src/main/java/com/epam/training/designpatterns/fastfoodrestaurant/RestaurantSimulator.java

```diff
@@ -0,0 +1,73 @@
+package com.epam.training.designpatterns.fastfoodrestaurant;
+
+import java.util.Random;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import com.epam.training.designpatterns.fastfoodrestaurant.entities.Client;
+import com.epam.training.designpatterns.fastfoodrestaurant.entities.Order;
+import com.epam.training.designpatterns.fastfoodrestaurant.food.Chips;
+import com.epam.training.designpatterns.fastfoodrestaurant.food.Condiment;
+import com.epam.training.designpatterns.fastfoodrestaurant.food.Food;
+import com.epam.training.designpatterns.fastfoodrestaurant.food.Hotdog;
+import com.epam.training.designpatterns.fastfoodrestaurant.food.Ketchup;
+import com.epam.training.designpatterns.fastfoodrestaurant.food.Mustard;
+import com.epam.training.designpatterns.fastfoodrestaurant.strategies.AmericanStyleCookingStrategy;
+import com.epam.training.designpatterns.fastfoodrestaurant.strategies.CookingStrategy;
+import com.epam.training.designpatterns.fastfoodrestaurant.strategies.JapaneseStyleCookingStrategy;
+import com.epam.training.designpatterns.fastfoodrestaurant.workstations.Chef;
+import com.epam.training.designpatterns.fastfoodrestaurant.workstations.DeliveryQueue;
+import com.epam.training.designpatterns.fastfoodrestaurant.workstations.OrderQueue;
+import com.epam.training.designpatterns.fastfoodrestaurant.workstations.Waiter;
+
+public class RestaurantSimulator {
+
+	public static final Random random = new Random();
+	
+	private OrderQueue orderQueue = new OrderQueue();
+	private DeliveryQueue deliveryQueue = new DeliveryQueue();
+	private Chef chef = new Chef(orderQueue, deliveryQueue, new AmericanStyleCookingStrategy());
+	private Waiter waiter = new Waiter(orderQueue, deliveryQueue);
+	
+	
+	public static void main(String[] args) throws InterruptedException {
+		RestaurantSimulator restaurant = new RestaurantSimulator();
+		restaurant.startSimulation(20, 4000);
```

Comment:
Magic numbers. Légyszi emeld ki őket így:
`private static final in MAX_NUMBER_OF_CLIENTS = 20;`
`private static final in MAX_SPAWN_TIME = 4000;`

Reply:
Igen, jogos! Sőt, ezeket ki fogom tenni properties file-ba inkább.